content/recipes/swagger.md

```diff
@@ -44,31 +44,27 @@ async function bootstrap() {
 
-In order to create a full document (with defined HTTP routes) we use the `createDocument()` method of the `SwaggerModule` class. This method takes two arguments, the application instance and the base Swagger options respectively.
-
-The last step is to call `setup()`. It accepts sequentially **(1)** path to mount the Swagger, **(2)** application instance, and **(3)** the document that describes the Nest application.
+Once we create a document, we can call `setup()` method. It accepts a **(1)** path to mount Swagger UI, **(2)** an application instance, and **(3)** the document that describes an API.
 
 Now you can run the following command to start the HTTP server:
 
 ```bash
 $ npm run start
 ```
 
-While the application is running, open your browser and navigate to `http://localhost:3000/api`. You should see a similar page:
+While the application is running, open your browser and navigate to `http://localhost:3000/api`. You should see Swagger UI.
 
 <figure><img src="/assets/swagger1.png" /></figure>
 
-The `SwaggerModule` automatically reflects all of your endpoints. In the background, it's making use of `swagger-ui-express` and creates a live documentation.
-
-> info **Hint** If you want to download the corresponding Swagger JSON file, you can simply call `http://localhost:3000/api-json` in your browser (if your Swagger documentation is published under `http://localhost:3000/api`).
+As you may noticed, the `SwaggerModule` automatically reflects all of your endpoints. Also, in order to display the Swagger UI, `@nestjs/swagger` makes use of either `swagger-ui-express` or `fastify-swagger` depending on the platform.
 
-#### Body, query, path parameters
+> info **Hint** If you want to download generated Swagger JSON file, you can navigate to `http://localhost:3000/api-json` in your browser (assuming that your Swagger documentation is available under `http://localhost:3000/api`).
 
-During the examination of the defined controllers, the `SwaggerModule` is looking for all used `@Body()`, `@Query()`, and `@Param()` decorators in the route handlers. Hence, the valid document can be created.
+#### Route parameters
 
-Moreover, the module creates the **models definitions** by taking advantage of the reflection. Take a look at the following code:
+When inspecting defined controllers, the `SwaggerModule` searches for all `@Body()`, `@Query()`, and `@Param()` decorators in route handlers. Hence, the valid document can be created. In addition, the module creates corresponding **models definitions** by taking advantage of the reflection. Look at the following code:
```

Comment:
The SwaggerModule searches for all @Body(), @Query(), and @Param() decorators in route handlers to generate the API document.  It also creates corresponding model definitions by taking advantage of reflection. Consider the following code: